src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -122,6 +122,9 @@ public enum Mixins {
     SPEEDUP_GRASS_BLOCK_RANDOM_TICKING(new Builder("Speed up grass block random ticking").setPhase(Phase.EARLY)
             .addMixinClasses("minecraft.MixinBlockGrass").addTargetedMod(TargetedMod.VANILLA).setSide(Side.BOTH)
             .setApplyIf(() -> SpeedupsConfig.speedupGrassBlockRandomTicking)),
+    FIX_REED_PLACING(new Builder("Fix placement of ItemReed").setPhase(Phase.EARLY)
+            .addMixinClasses("minecraft.MixinItemReed").addTargetedMod(TargetedMod.VANILLA).setSide(Side.BOTH)
+            .setApplyIf(() -> SpeedupsConfig.speedupGrassBlockRandomTicking)),
```

Comment:
Should this be in the fixes section rather than speedup?

Reply:
also, why are you gating this behind SpeedupsConfig.speedupGrassBlockRandomTicking rather than its own setting?

Reply:
For both of these: Because I copy pasted the first other thing I could see relating to grassy vanilla stuff to get it working and didn't really look back at the details once it did work. Thanks for catching this.